include/dynamatic/Dialect/Handshake/HandshakeArithOps.td

```diff
@@ -101,6 +101,43 @@ class Handshake_Arith_IToICastOp<string mnemonic, list<Trait> traits = []> :
   let hasVerifier = 1;
 }
 
+class Handshake_Arith_FToFCastOp<string mnemonic, list<Trait> traits = []> :
+  Handshake_Arith_Op<mnemonic, traits # [SameExtraSignalsInterface,
+                                         ReshapableChannelsInterface]
+> {
+  let arguments = (ins IntChannelType:$in);
```

Comment:
Both the argument and result should be `FloatChannelType` no?

Reply:
Thanks for the catch!